### PR TITLE
Updated groups to eliminate unknown

### PR DESCRIPTION
**Pull request by maxsimov**

### CODE REVIEW

File: launchpad_report/config.yaml [hunk 1]
```diff
@@ -40,11 +40,17 @@ teams:
     sbogatkin
     xdeller
     andreika-mail
-  manashkin:
+    omolchanov
+    bpiotrowski
+    sgolovatiuk
+    bogdando
+    smakar
+ manashkin:
```
Comment (dmi-try) [inline launchpad_report/config.yaml]: You've lost one space here. It breaks yaml

File: launchpad_report/config.yaml [hunk 2]
```diff
@@ -71,13 +81,15 @@ teams:
     acharykov
     skulanov
     corepb
+    mmatuszkowiak
+    ibelikov
   dborodaenko:
     dborodaenko
     rmoe
     xarses
     dreidellhasa
```
Comment (dmi-try) [inline launchpad_report/config.yaml]: Meg is in doc's team, I guess.

File: launchpad_report/config.yaml [hunk 3]
```diff
@@ -71,13 +81,15 @@ teams:
...
     docaedo
```
Comment (dmi-try) [inline launchpad_report/config.yaml]: Christopher Aedo is in docs team, but I'm not sure

File: launchpad_report/config.yaml (hunk 2 above)
Comment (maxsimov) [inline launchpad_report/config.yaml]: Meghan Moe? looks like she is in Administration

Comment (dmi-try) [inline launchpad_report/config.yaml]: Meg McRoberts

Comment (maxsimov) [inline launchpad_report/config.yaml]: I don't know, we will move later when we understand this

File: launchpad_report/config.yaml (hunk 3 above)
Comment (maxsimov) [inline launchpad_report/config.yaml]: also will check/correct later

File: launchpad_report/config.yaml (hunk 2 above)
Comment (dmi-try) [inline launchpad_report/config.yaml]: I'm sure she is a techwriter

**Pull request merged**